apiproxy/proxy.go

```diff
@@ -56,7 +54,7 @@ func (p *proxy) Launch() error {
 	router.PathPrefix("/graphql").Handler(p.dgraphqlProxy)
 	router.PathPrefix("/graphiql").Handler(p.dgraphqlProxy)
 	router.PathPrefix("/v1/chain/push_transaction").Handler(p.eoswsProxy)
-	router.PathPrefix("/v1/chain").Handler(p.nodeosProxy)
+	router.PathPrefix("/v1/chain").Handler(p.eoswsProxy)
```

Comment:
Now we do real double proxying :) But well, if it works! I imagine you tested all this, or would you prefer that we give it a go?

Reply:
@maoueh I tested it during the hackathon but we might want to fix it elsewhere/differently. I would appreciate if you gave it a go as I believe it should be part of the dev experience... How to configure client-js for a local `dfuseeos` etc.